<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>171</width>
         <height>401</height>
        </rect>
       </property>
       <property name="title">
        <string>随机数</string>
       </property>
       <widget class="QListWidget" name="randomList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>151</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>50</y>
         <width>171</width>
         <height>401</height>
        </rect>
       </property>
       <property name="title">
        <string>快速排序</string>
       </property>
       <widget class="QListWidget" name="quickList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>151</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>171</width>
         <height>401</height>
        </rect>
       </property>
       <property name="title">
        <string>冒泡排序</string>
       </property>
       <widget class="QListWidget" name="bubbleList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>151</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="startBtn">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>470</y>
         <width>221</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
